ginga/misc/plugins/SaveImage.py

```diff
@@ -169,14 +172,14 @@ def redo(self, *args):
 
         # Only list modified images for saving. Scanning Datasrc is enough.
         if mod_only:
-            func = self.keys_from_datasrc
+            all_keys = channel.datasrc.sortedkeys
```

Comment:
Ok for now, but it would be good to use an API on the `Datasrc` since its internal implementation could easily change.